lib/output.js

```diff
@@ -88,13 +88,14 @@ function OutputStream(options) {
 
     function make_string(str, quote) {
         var dq = 0, sq = 0;
-        str = str.replace(/[\\\b\f\n\r\t\x22\x27\u2028\u2029\0\ufeff]/g, function(s){
+        str = str.replace(/[\\\b\f\n\r\v\t\x22\x27\u2028\u2029\0\ufeff]/g, function(s){
             switch (s) {
               case "\\": return "\\\\";
               case "\b": return "\\b";
               case "\f": return "\\f";
               case "\n": return "\\n";
               case "\r": return "\\r";
+              case "\v": return "\\v";
```

Comment:
Do we care that this works in old/broken environments? If so, this should be `"\\x0B"` instead of `"\\v"`. See https://github.com/mishoo/UglifyJS2/issues/6.